packages/google_maps_flutter/google_maps_flutter/example/lib/main.dart

```diff
@@ -75,6 +79,36 @@ void main() {
       GoogleMapsFlutterPlatform.instance;
   if (mapsImplementation is GoogleMapsFlutterAndroid) {
     mapsImplementation.useAndroidViewSurface = true;
+    initializeMapRenderer();
   }
   runApp(const MaterialApp(home: MapsDemo()));
 }
+
+Completer<AndroidMapRenderer?>? _initializedRendererCompleter;
+
+/// Initializes map renderer to with `latest` renderer type for Android platform.
+/// The renderer must be requested before creating GoogleMap instances,
```

Comment:
Blank line above this, per Effective Dart guidance about having a single-sentence first paragraph.

Reply:
Changed as requested